src/tools/ci/docker/linux_fedora_rawhide

```diff
@@ -0,0 +1,49 @@
+FROM fedora:rawhide
+RUN \
+  yum -y update && \
+  yum -y install \
+  git \
+  meson \
+  perl \
+  perl-IPC-Run \
+  \
+  bison \
+  ccache \
+  clang \
+  flex \
+  gcc \
+  gettext \
+  lz4 \
+  ninja-build \
+  \
+  libicu-devel \
+  libuuid-devel \
+  libxml2-devel \
+  libxslt-devel \
+  llvm-devel \
+  lz4-devel \
+  pam-devel \
+  python3-devel \
+  readline-devel \
+  systemd-devel \
+  tcl-devel \
+  \
+  openldap-devel \
+  openldap-servers \
+  openldap-clients \
+  \
+  openssl \
+  openssl-devel \
+  \
+  krb5-devel \
+  krb5-server \
+  krb5-server-ldap \
+  krb5-workstation \
```

Comment:
It's not exactly obvious, but what I've tried to group so far are different types of dependencies. -devel are headers and needed for compilation. Whereas e.g. the openldap server is only needed to run run tests etc.

Reply:
Done.